AlumniConnect/views.py

```diff
@@ -38,6 +39,39 @@ class LoginFormView(SuccessMessageMixin, LoginView):
     # success_url = '/'
     success_message = "Logged in successfully!"
 
+def my_login(request):
+    """
+     Login user only if user is also verified by admin
+    """
+    # checking if user is already logged in
+    if request.user and request.user.is_authenticated:
+        return redirect('home')
+    
+    if request.method == 'POST':
+        form = AuthenticationForm(request,data = request.POST)
+        if form.is_valid():
+            username = form.cleaned_data.get('username')
+            user = User.objects.get(username = username)
+            
+            # if user is admin no need to check other thing
+            if user.is_staff:
```

Comment:
`user.is_superuser`?

We might give some access to some students, but they'd still need to have complete profile.